docs/developer_guide.md

```diff
@@ -0,0 +1,140 @@
+
+All of our code is open sourced. Developers can follow this guide to get projects set up.
+
+### Prerequisite
+
+1. Download `qtum-x.xx.xx-osx64.tar.gz`  from [https://github.com/qtumproject/qtum/releases](https://github.com/qtumproject/qtum/releases) and unzip it.
+
+2. Then open terminal and enter the directory:
+      
+         $ cd /qtum-x.xx.xx/bin
+
+3. Run command to start sync.
+
+         $ sudo ./qtumd -testnet -rpcuser=bodhi -rpcpassword=bodhi -logevents -printtoconsole      
+
+      The terminal should show some info like below
+
+      ![img](img/sync_screenshot.png)
+
+4. Open another terminal and cd into the same directory, run command
+
+         $ ./qtum-cli -rpcuser=bodhi -rpcpassword=bodhi -rpcport=13889 getblockcount       
```

Comment:
`./qtum-cli -rpcuser=bodhi -rpcpassword=bodhi -testnet getblockcount`? We use the default port